docs/UserGuide.md

```diff
@@ -15,30 +19,43 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
 ## Quick start
 
-1. Ensure you have Java `17` or above installed in your Computer.
+1. Ensure you have Java `17` or above installed in your computer.
+- To install Java 17, download Java [here](https://www.oracle.com/java/technologies/javase/jdk17-archive-downloads.html).
+- To check your Java version, refer [here](https://www.java.com/en/download/help/version_manual.html#).
 
-1. Download the latest `.jar` file from [here](https://github.com/se-edu/addressbook-level3/releases).
+2. Download our latest release file from [here](https://github.com/se-edu/addressbook-level3/releases). It is a `.jar` file.
```

Comment:
Should we include an external link to explain what is a .jar file?

Reply:
Same as above, external link?

Reply:
Yup